### PR TITLE
Updated GML.js to 2023.1 version

### PR DESCRIPTION
**Pull request by gnysek**
Updated GML language data to most up-to-date version, based on 2023.100.0.264 runtime (January 2023 release), with many features added in recent months. Many functions in current definition file were outdated (they were marked as deprecated for around 5-10 years or even already removed in recent months and years).

### Changes
Generated from GameMaker runtime internal `fnames` file, with rule:
- `KEYWORDS`: manually, as they are built-in in IDE and not mentioned in `fnames` file
- `BUILT_INS`: functions; everything that have `[a-z_]+\(.*?\)` in definition (both UK and US spelling)
- `SYMBOLS`: macros; everything that ended with `#` in `fnames`
- `LANGUAGE_VARIABLES`: various global/local variables (reserved words); everything that ended by `*` (readonly), `@` (instance variable), or didn't ended with `(...)`
That exact `fnames` file can be found here: https://gist.github.com/gnysek/a69ec324df61131b2546d1580f29a760

### Checklist
- [x] Added markup tests, or they don't apply here because... (only keyword updates)
- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by joshgoebel** [comment]
Please explain removal of LITERALS.

**Comment by gnysek** [comment]
I moved then to SYMBOLS, where other GM literals were already. All of those are generally macros which are converted (usually) to integers on compile time.

File: src/languages/gml.js [hunk 1]
```diff
@@ -2599,8 +2774,7 @@ export default function(hljs) {
     "vk_subtract",
     "vk_tab",
     "vk_up"
-  ];
-  const LANGUAGE_VARIABLES = [
+  const LANGUAGE_VARIABLES = [];
     "alarm",
     "application_surface",
     "argument",
```
Comment (joshgoebel) [inline src/languages/gml.js:2780]: Not sure how tests ever passed? This doesn't even look syntactically correct?  Did it get warped somehow?

Comment (gnysek) [reply src/languages/gml.js]: It was auto-generated, seems that at some point I overlooked that it started to generate invalid JS code. Fixed now.

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**